spring-cloud-stream-schema-server/src/test/java/org/springframework/cloud/stream/schema/server/SchemaRegistryServerAvroTests.java

```diff
@@ -45,7 +45,7 @@
  */
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT,
-		properties = {"--spring.main.allow-bean-definition-overriding=true"})
+				properties = "spring.main.allow-bean-definition-overriding=true")
```

Comment:
Is there anything what really makes us to keep this `allow-bean-definition-overriding` here?

I believe the point of this PR to get rid of it...

Thanks